src/protocols/aeternity/views/AccountCreateModal.vue

```diff
@@ -39,18 +39,36 @@
         :disabled="!isOnline"
         @click="createMultisigAccount()"
       />
+      <BtnSubheader
```

Comment:
There is already a button with `v-if="!isMultisig"` above. I suggest to group them:
```
<template v-if="isMultisig">
  // MS BTN
</template>
<template v-else>
  // NON MS BTNs
</template>
```

Reply:
This would change the order of the buttons, because the button in the middle is shown both for `if multisig` & `if !multisig`. There's no `v-else`